Dockerfile

```diff
@@ -27,6 +30,12 @@ ADD package.json /app/
 # to allow forcing rebuilds from this stage
 ARG REBUILD
 
+# Download and format ad host blocklist as JSON
+RUN mkdir -p /tmp/ads && cd /tmp/ads && \
```

Comment:
Great! For now, lets keep this here rather than base image, as that way can update the block list more quickly.